admix/manager.py

```diff
@@ -397,3 +400,65 @@ def clean_local_dir(path, before_straxen_version, ensure_rucio=False, dry_run=Tr
         print(f"Files that we can delete written to {tmpfile}")
 
 
+def bring_online(did,rse):
+    print("Bringing online {0} from {1}".format(did,rse))
+
+    if rse not in ['SURFSARA_USERDISK','CNAF_TAPE3_USERDISK','CNAF_TAPE2_USERDISK','CNAF_TAPE_USERDISK','CCIN2P3_USERDISK']:
```

Comment:
Better to put something like
```python
TAPE_RSES = ['SURFSARA_USERDISK','CNAF_TAPE3_USERDISK','CNAF_TAPE2_USERDISK','CNAF_TAPE_USERDISK','CCIN2P3_USERDISK']
```
in top of the script for better maintenance of hardcoded configs.

Reply:
Also what about?
```
| NIKHEF2_USERDISK    | 421.270 TB | 400.000 TB | 0.000 B      |
| NIKHEF_USERDISK     | 711.558 GB | 275.000 TB | 274.288 TB   |
```
I forgot if they are mixed or tapes...

Reply:
I will make the change.
NIKHEF and NIKHEF2 are 100% disk (but, careful, NIKHEF should never be used, the numbers there are just what rucio has in the catalogue but the physical storage has been dismissed)